source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -6,11 +7,11 @@
 #include <cstring>
 #include <random>
 #include <stdexcept>
+#include <vector>
 
 #include "device_server.h"
 #include "enumerate_devices.h"
 #include "nidaqmx/nidaqmx_client.h"
-#include "nidaqmx/nidaqmx_service.h"
```

Comment:
Note: tests don't need to include the service because that's all internal to the `register_all_services` implementation. It doesn't make a huge difference because the `client.h` files include `grpc.pb.h` (needed for request/response messages and the Stub). If we have issues with long-pole test files, we can try to address that. The easiest improvement would be encapsulating the `stub` so that we only need `pb.h` (unfortunately the stub is a nested class, so we can't just forward declare it ...).